packages/mui-system/src/cssVars/getInitColorSchemeScript.tsx

```diff
@@ -5,15 +5,15 @@ export const DEFAULT_COLOR_SCHEME_STORAGE_KEY = 'mui-color-scheme';
 export const DEFAULT_ATTRIBUTE = 'data-mui-color-scheme';
 
 export default function getInitColorSchemeScript(options?: {
-  defaultMode?: 'light' | 'dark' | 'system';
+  enableSystem?: boolean;
```

Comment:
Just for my understanding, what does this boolean represent?

Reply:
This script is run before the browser render HTML. By specifying `enableSystem: true`, it will set the mode to user preference.

Reply:
'light' | 'dark' | 'system' does not make sense because it should be just true or false.